### ISSUE TITLE
Accessibility: Table commands are not working (Caps+F3.,Caps+F4 ) to navigate within the table

### ISSUE THREAD

**Issue by Lovakumar** [OfficeDev/office-ui-fabric-react#3835]
Navigate to VssDetailsList and use table commands (For Ex: Caps+F3, Caps+F4 to navigate to next cell in row and column).

Platform:
Microsoft Edge 41.16299.15.0
Microsoft EdgeHTML 16.16299
OS Version : 1709
OS build : 16299.125

**Comment by battletoilet** [comment]
Hi @lovakumar, we don't support these commands in the DetailsList. We just support navigation arrow keys. Can you please provide more details about the issue?

**Comment by lovakumar** [comment]
@battletoilet this issue was reported by our accessibility testing team. I check with them again and get back to you on whether they are ok with arrow keys or not.

**Comment by lovakumar** [comment]
Here are the details I have got from our bug basher:

We are able to access all the content in the table using Caps+arrow keys and narrator announces the column headers properly. But Narrator announces incorrect row number. Ex, Narrator announces as Row 2 of 36 for all the table rows.
Entire column details can be obtained using Caps+F7 key and Caps+F4 can be used to read the next column cell, the same cannot be retrieved when navigating using arrow keys. 
But few of the table commands such as Caps+F8 to read the entire row and Caps+F9, Caps+F10 to read the  column header and row header works fine for this table.

**Comment by caradong** [comment]
@lovakumar thank you for reporting this. We received a very similar bug by an internal accessibility test team. Just to provide a bit more information, here is what the actual/expected behavior for caps + F3, caps + F4.

caps + F3 focus should shift to the next cell in a row, actual: focus is not shifted
caps + F4 focus should shift to the next cell in the same column, actual: focus is shifted to the first column in the next row

**Comment by Jahnp** [comment]
@betrue-final-final -- can you look at this when you have a chance? Do you know offhand if these are commands we actively support in other areas? 

Also, given the component called out here is `VssDetailsList`, I'm also wondering @lovakumar if this is something that should be implemented locally in your scenario. 

@aditima for visibility (sorry for all the DetailsList bugs coming your way!)

**Comment by betrue-final-final** [comment]
This is definitely something we should support, and it looks like we only do partway. I'm not sure there are any other applicable areas in Fabric, excepting for maybe DatePicker.

As this is functionality called out specifically by Narrator, we will want to support it.

**Comment by caradong** [comment]
@betrue-final-final is there any update on this issue?

**Comment by dzearing** [comment]
Moving off aditi; Kevin as you're looking at DetailsList accessibility, is this a dupe?

**Comment by cschlechty** [comment]
As far as I know, table commands are working as expected. Given the age of this, I'm closing this issue.
